app/routes.py

```diff
@@ -1,2 +1,231 @@
-from flask import Blueprint
+from flask import Blueprint, request, make_response, jsonify
+from app import db
+from app.models.task import Task
+from app.models.goal import Goal
+from datetime import datetime
+import os
+from dotenv import load_dotenv
 
+tasks_bp = Blueprint("tasks", __name__, url_prefix="/tasks")
+
+
+@tasks_bp.route("", methods=["POST","GET"])
+def handle_tasks():
+    if request.method=="POST":
+        request_body = request.get_json()
+        if "completed_at" not in request_body.keys() or "description" not in request_body.keys() or "title" not in request_body.keys():
```

Comment:
You do not have to use keys() here you could just say `if not "title" in request_body` etc.